ndarray-rand/src/lib.rs

```diff
@@ -225,17 +226,93 @@ where
         R: Rng + ?Sized,
         A: Copy,
         D: RemoveAxis;
+
+    /// Shuffle `self`'s slices along `axis`.
```

Comment:
Shuffle the array's xyz along `axis`.

We need a name for it, not sure if it should be slices. For the `.lanes(axis)` method we call them lanes - but those are perpendicular to `axis` in that case.

Reply:
I'd be tempted to call them *planes*. We shuffle among the items in an AxisIter, right? Each such item is *n - 1* dimensional in an ndarray.

Reply:
An alternative name could be _projections_, which has the plus of not carrying an intuitive dimensionality along with it (I think 2D when I read planes).